packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -52,6 +53,10 @@ const threadableLoaders = [
 
 // TODO: refactor if the interface pattern is used a total of 3 times
 
+export const users = () => {
+  return new ProxiedCache('User')
```

Comment:
this lets folks still call `dataLoader.get('users').load('user123')`. but now, it will attempt to fetch from the cache instead of going directly to the DB